sdks/python/apache_beam/io/filebasedsink.py

```diff
@@ -68,6 +68,9 @@ def __init__(
       shard_name_template=None,
       mime_type='application/octet-stream',
       compression_type=CompressionTypes.AUTO,
+      *,
```

Comment:
Do we have code style guide about the usage of asterisk in function parameters?

Reply:
I don't think we have any guidance here (other than that which is generic to Python, which would indicate most of these arguments should be passed by keyword).